packaging/build_deb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Builds the libsxg deb packages. The script must be run at the root
+# directory of the libsxg source tree. The deb packages will be copied to
+# the current directory.
+#
+# This script is written mainly for CI.
+
+set -ex
+
+readonly SRCDIR="${PWD}"
+readonly WORKDIR="$(mktemp -d --tmpdir=/tmp build_deb.XXXXXX)"
+readonly NAME='libsxg'
+readonly VERSION='0.1'
+readonly TARNAME="${NAME}_${VERSION}"
+readonly TARGET_NAME='libsxg-dev'
+readonly ARCHITECTURE="$(dpkg --print-architecture)"
+
+echo "Working in ${WORKDIR}"
+
+git archive --format=tar.gz --output="${WORKDIR}/${TARNAME}.orig.tar.gz" \
+    --prefix="${TARNAME}/" HEAD
```

Comment:
Optional: `${NAME}` or `${NAME}-${VERSION}`.

Reply:
OK, changed it to `${NAME}`.